modules/fff/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_FFF)
+
+  set(FFF_DIR ${ZEPHYR_CURRENT_MODULE_DIR})
+
+  zephyr_library()
+
+  zephyr_include_directories(${FFF_DIR})
```

Comment:
I'd just write it out in one line like:
```cmake
zephyr_include_directories(${ZEPHYR_CURRENT_MODULE_DIR})
```
But I'm don't know there is a standard we should follow or not.

Reply:
Done